platform_versioned_docs/version-23.4.0/functionality_matrix/functionality_matrix.mdx

```diff
@@ -7,12 +7,13 @@ tags: [compatibility, nextflow, nf-launcher]
 
 Each Seqera Platform version makes use of `nf-launcher` to determine the Nextflow version used as its baseline. This Nextflow version can be overridden with the `NXF_VER` environment variable in your `nextflow.config` file, but note that Seqera may not work reliably with Nextflow versions other than the baseline version.
```

Comment:
```suggestion
Each Seqera Platform version makes use of `nf-launcher` to determine the Nextflow version used as its baseline.
```

Duplicated in the section below.

It's also wrong - can't set this environment variable within a `nextflow.config` file, or at least it has no effect as Nextflow has already launched by the time the config file is parsed.